modules/targetVideoBidAdapter.js

```diff
@@ -0,0 +1,179 @@
+import find from 'core-js-pure/features/array/find.js';
+import { getBidRequest } from '../src/utils.js';
+import { VIDEO } from '../src/mediaTypes.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+
+const SOURCE = 'pbjs';
+const BIDDER_CODE = 'targetVideo';
+const ENDPOINT_URL = 'https://ib.adnxs.com/ut/v3/prebid';
+const MARGIN = 1.35;
+
+export const spec = {
```

Comment:
Add the `supportedMediaTypes` property to `spec`. Which ones does this adapter support?

Reply:
Added.